packages/react-reconciler/src/__tests__/ReactHooks-test.internal.js

```diff
@@ -1696,17 +1886,20 @@ describe('ReactHooks', () => {
       return null;
     }
 
-    expect(() => {
-      ReactTestRenderer.create(
-        <>
-          <A />
-          <B />
-        </>,
-      );
-    }).toThrow('Hello');
+    await expect(async () => {
+      await act(() => {
+        ReactTestRenderer.create(
+          <>
+            <A />
+            <B />
+          </>,
+          {unstable_isConcurrent: true},
+        );
+      });
+    }).rejects.toThrow('Hello');
 
     if (__DEV__) {
-      expect(console.error).toHaveBeenCalledTimes(2);
+      expect(jest.mocked(console.error)).toHaveBeenCalledTimes(2);
```

Comment:
As far as I know [`jest.mocked`](https://jestjs.io/docs/mock-function-api#jestmockedsource-options) is only a helper for TypeScript usage and doesn't do anything at runtime

Reply:
I was getting errors related to spy usage and thought adding this fixed them. But now the tests also work without the changes so it was probably related to some other change in the end. Will remove